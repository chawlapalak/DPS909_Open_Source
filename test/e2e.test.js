const { run } = require('./run');

describe('end-to-end integration', () => {
  test('when argument --help is specified should print help message', async () => {
    const { stderr, stdout, exitCode } = await run('--help');
    expect(exitCode).toBe(0);
    expect(stdout).toMatchSnapshot();
    expect(stderr).toEqual('');
  });
});