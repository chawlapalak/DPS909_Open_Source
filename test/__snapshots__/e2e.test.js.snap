// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`end-to-end integration when argument --help is specified should print help message 1`] = `
"Usage: index [options]

Options:
  -v, -V, --version     prints tool version
  -f, --file <path>     Path to file
  --all                 All URLs
  --good                Good URLs
  --bad                 Bad URLs
  -i, --ignore <path>   Path to file with URLs to ignore
  -e, --endpoint <url>  URL to fetch post list
  -h, --help            display help for command"
`;
